### PR TITLE
Added new utilities to split a .wav file and fixed the bugs in vtt-json package

### PR DESCRIPTION
**Pull request by jmkl009**

### CODE REVIEW

File: modules/vtt-json.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+function convertVttToJson(vttString) {
+  return new Promise((resolve, reject) => {
```
Comment (chirantan31) [inline modules/vtt-json.js]: Why is this in the form of a Promise, are you using any asynchronous code?

File: modules/split_wav.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+// (async() => {
+//   var vtt = fs.readFileSync('sample.vtt').toString();
+//   var json = await vttToJson(vtt);
+//   // console.log(json);
+//   var ret = await splitWavParallel('sample.wav', json);
+//   console.log(ret);
+// })();
+
+const {spawn} = require('child_process');
```
Comment (chirantan31) [inline modules/split_wav.js]: Could you use the promisified version of this library?

File: modules/vtt-json.js (hunk 1 above)
Comment (jmkl009) [reply modules/vtt-json.js]: The original library defines the function this way, do you want me to go ahead and make it synchronous?

File: modules/split_wav.js (hunk 2 above)
Comment (jmkl009) [reply modules/split_wav.js]: Using the promisified version gives over my control of error logging.

File: modules/vtt-json.js (hunk 1 above)
Comment (chirantan31) [reply modules/vtt-json.js]: Yeah you could do that, I don't see a need for this to be asynchronous

File: modules/split_wav.js (hunk 2 above)
Comment (chirantan31) [reply modules/split_wav.js]: https://www.npmjs.com/package/child-process-promise
Could you elaborate further on what logging will this library prevent you from doing?

Comment (jmkl009) [reply modules/split_wav.js]: There are two ways the process throws an error, one is the process was not created correctly, and one is the process exits non-zero. I will not be able to deal with the case where the process exits non-zero.

Comment (chirantan31) [reply modules/split_wav.js]:
```
const { spawn } = require('child-process-promise');
        const ffmpeg = spawn('ffmpeg', ['-nostdin', '-i', pathToFile, '-c:a', 'pcm_s16le', '-ac', '1', '-y', '-ar', '16000', outputFile]);
        var res = await ffmpeg;
        console.log(res.childProcess.exitCode);
```
If your issue is with accessing the exitCode, you could do as specified above, 
Try doing a `console.log(res.childProcess)` there are more details available in that object

**Pull request merged**